### DISCUSSION TITLE
How to get the user ID from the JWT?

### DISCUSSION THREAD

**Discussion by zaiqsaysa** [prest/prest#836] [API Server] [answered]
Hi! I'm new to prestd. I have created_by and modified_by fields in my table and I want it to be automatically populated or updated based on certain conditions, such as user authentication. Is it possible to do this in prestd?

**Comment by arxdsilva** [comment, marked as answer]
Hi @zaiqsaysa, you'll need to do a custom route for that unless you pass it as part of your request body. Please see this documentation section for how to create the custom route: https://docs.prestd.com/plugins/endpoint-plugin
